control-plane/subcommand/server-acl-init/command.go

```diff
@@ -139,12 +139,11 @@ func (c *Command) init() {
 	var unused bool
 	c.flags.BoolVar(&unused, "create-inject-namespace-token", false,
 		"Toggle for creating a connect injector token. Only required when namespaces are enabled. "+
-			"Deprecated: set -enable-namespaces and -create-inject-token instead.")
-
-	c.flags.BoolVar(&c.flagCreateInjectToken, "create-inject-auth-method", false,
-		"Toggle for creating a connect inject auth method. Deprecated: use -create-inject-token instead.")
-	c.flags.BoolVar(&c.flagCreateInjectToken, "create-inject-token", false,
-		"Toggle for creating a connect inject auth method and an ACL token.")
+			"Deprecated: set -enable-namespaces and -create-inject-policy instead.")
+	c.flags.BoolVar(&c.flagCreateInjectPolicy, "create-inject-auth-method", false,
+		"Toggle for creating a connect inject auth method. Deprecated: use -create-inject-policy instead.")
```

Comment:
same here: we prob can remove the deprecated flag since probably no one can run helm chart with that version of consul-k8s anyway